app/helpers/users_helper.rb

```diff
@@ -1,2 +1,5 @@
 module UsersHelper
+  def find_fp(fp_id)
+    @fp_id = FinancialPlanner.find_by(id: fp_id)
```

Comment:
あれ、ヘルパーでインスタンス変数に代入するとコントローラとか view で参照できるんだろうか 🤔 
参照できたとしてもコントローラ・ヘルパーでインスタンス変数が分散してしまうので、↓にして呼び出し元でインスタンス変数に代入してあげたほうがいいかなと思いました！

```suggestion
    FinancialPlanner.find_by(id: fp_id)
```

Reply:
参照自体はできそうです！ここでは返り値をそのまま用いているだけなので変数にする必要がなかったです！ありがとうございます！